client/components/domains/domain-search-results/index.jsx

```diff
@@ -55,6 +55,7 @@ class DomainSearchResults extends Component {
 		pendingCheckSuggestion: PropTypes.object,
 		unavailableDomains: PropTypes.array,
 		domainAndPlanUpsellFlow: PropTypes.bool,
+		useProvidedProductsList: PropTypes.bool,
```

Comment:
Since we are checking that we indeed have a truthy non-empty products list do we still need of these `useProvidedProductsList` props?

Reply:
I wanted to make sure I'm not breaking any existing thing here, better safe than sorry, but I can remove it if it's not necessary